app/swagger/swagger/requests/travel_pay.rb

```diff
@@ -73,11 +73,15 @@ class TravelPay
           parameter :authorization
 
           parameter do
-            key :name, 'appt_datetime'
-            key :in, :query
-            key :description, 'Appointment claim submission datetime'
-            key :required, false
-            key :type, :string
+            key :name, :appointmentDatetime
+            key :in, :body
+            key :description, 'Appointment claim start time'
+            key :required, true
+            schema do
+              property :appointmentDatetime do
+                key :type, :string
+              end
+            end
```

Comment:
Thanks for updating this. I'd just started to do some changes locally to fix my doc updates here but better to get it in one fell swoop.